src/pybind/mgr/dashboard/frontend/src/app/shared/directives/update-scope.directive.ts

```diff
@@ -0,0 +1,34 @@
+import { AfterViewInit, Directive, ElementRef, Input, Optional } from '@angular/core';
+import { Permissions } from '../models/permissions';
+import { AuthStorageService } from '../services/auth-storage.service';
+import { PermissionScopeDirective } from './permission-scope.directive';
+
+@Directive({
+  selector: 'input:not([cdNoUpdateScope]), select:not([cdNoUpdateScope]), [cdUpdateScope]'
+})
+export class UpdateScopeDirective implements AfterViewInit {
+  private permissions: Permissions;
+  private service_name: keyof Permissions;
+
+  @Input() disabled: boolean;
```

Comment:
not needed
```suggestion
```

Reply:
This disabled is a required one. There are some input fields which are by default has a disabled value. I dont want to change that values, so I grab that disabled value over here.